api/spec/spec_helper.rb

```diff
@@ -64,6 +64,12 @@
     Spree::Api::Config[:requires_authentication] = true
   end
 
+  config.include ActiveJob::TestHelper
+  config.after(:each) do
+    ActiveJob::Base.queue_adapter.enqueued_jobs = []
+    ActiveJob::Base.queue_adapter.performed_jobs = []
```

Comment:
:+1: I think you're right this isn't necessary anymore. I only needed it before including ActiveJob::TestHelper, which adds a setup step which clears this.